<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveDatabaseDialog</class>
 <widget class="QDialog" name="SaveDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New column name</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="saveMethodGb">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="saveMethodGbLayout">
      <item>
       <widget class="QRadioButton" name="customFormatRb">
        <property name="text">
         <string>Custom format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sqlitePageRb">
        <property name="text">
         <string>SQLite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mongoDbPageRb">
        <property name="text">
         <string>MongoDB</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="saveMethodGbSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="customFormatPage">
      <layout class="QHBoxLayout" name="customFormatPageLayout">
       <item>
        <widget class="QLabel" name="customFilePathLb">
         <property name="text">
          <string>Save path</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="customFormatPageSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>265</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="selectFolderBtn">
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sqlitePage">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>SQLite</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mongoDbPage">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>MongoDb</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBoxLayout">
     <item>
      <spacer name="buttonBoxSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
